project3.0/css/style.css

```diff
@@ -1,13 +1,464 @@
-.h2 {
-    position: absolute;
-    width: 506px;
-    height: 105px;
-    left: 148px;
-    top: 228px;
-    font-family: 'Heebo';
+:root {
+    --font-main: 'Heebo', sans-serif;
+    --color-primary: #21243D;
+    --color-secondary: #000000;
+}
+
+.header {
+    display: block;
+}
+
+.header-wrapper {
+    display: flex;
+    justify-content: flex-end;
+    margin-left: 60px;
+    margin-right: 60px;
+}
+
+.header-wrapper ul {
+    display: flex;
+    justify-content: space-around;
+    gap: 30px;
+    list-style-type: none;
+    text-decoration: none;
+}
+
+.header-wrapper li:active {
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.header-wrapper li:hover {
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.header-wrapper a {
+    text-decoration: none;
+    color: var(--color-secondary);
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 29px;
+}
+
+
+.section-info {
+    display: block;
+    margin-top: 140px;
+}
+
+.container {
+    max-width: 856px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.container-fluid {
+    max-width: 1152px;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+.section-info-wrapper {
+    display: flex;
+    justify-content: space-between;
+    gap: 115px;
+}
+
+
+.info-left h1 {
+    font-family: var(--font-main);
     font-style: normal;
     font-weight: 700;
     font-size: 44px;
     line-height: 60px;
-    color: #21243D
+    color: var(--color-primary);
+}
+
+.info-left p {
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: var(--color-primary);
+}
+
+.info-left button {
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 29px;
+    color: #FFFFFF;
+    padding: 13px 20px 13px 20px;
+    background: #FF6464;
+    border-radius: 2px;
+    border-color: #FF6464;
+}
+
+.section-posts {
+    display: block;
+    background: #EDF7FA;
+    margin-top: 71px;
+    padding-bottom: 32px;
+}
+
+.section-posts-wrapper {
+    display: flex;
+    flex-direction: column;
+}
+
+.post-item {
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+}
+
+.post-title {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.post-title h2 {
+    margin-top: 0;
+    margin-bottom: 0;
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 60px;
+    color: var(--color-primary);
+}
+
+.post-title a {
+    display: block;
+    padding: 24px 0px 24px 0px;
+    text-decoration: none;
+    font-family: var(--font-main);
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #00A8CC;
+}
+
+.single-post {
+    background-color: white;
```

Comment:
All inner elements have the same color, so you can remove color from .single-post h2,.single-post-date,  .single-post p.

And add it here color: var(--color-primary);